client/components/atoms/ArrowInput/ArrowInput.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from "react";
+import arrowIcon from "../../../assets/icons/icon_chevron-small-left.svg";
+import closeBtn from "../../../assets/icons/icon_close.svg";
+
+const ArrowInput = (props) => {
+  useEffect(() => {
+    return () => {};
```

Comment:
This is not necessary

Reply:
why is this still here ? it does nothing

Reply:
thats for the redux - prepared with states in the homepage component...